index.html

```diff
@@ -2194,23 +2194,26 @@ <h3>Data Schemas</h3>
         <h3>Refreshing</h3>
 
         <p>
-It is useful for systems to enable the manual or automatic refresh of a
+It is useful for systems to enable the manual or automatic refresh of an
 <a href="#expiration">expired</a> <a>verifiable credential</a>. This
 specification enables an <a>issuer</a> to include a link to a refresh service.
         </p>
         <p>
 The <a>issuer</a> can include the refresh service as an element inside the
 <a>verifiable credential</a> if it is intended for either the <a>verifier</a>
 or the <a>holder</a> (or both), or inside the <a>verifiable presentation</a>
-if it is intended for the <a>holder</a> only.
+if it is intended for the <a>holder</a> only. In the latter case, this enables 
+the <a>holder</a> to refresh the <a>verifiable credential</a> before creating 
+a <a>verifiable presentation</a> to share with a <a>verifier</a>. In the former case, 
+including the refresh service inside the <a>verifiable credential</a> enables both 
+the <a>holder</a> and the <a>verifier</a> to perform future updates of the credential.
         </p>
         <p>
-Including the refresh reference inside the <a>verifiable presentation</a>
-enables the <a>holder</a> to refresh the <a>credential</a> details before
-creating a <a>presentation</a> to share with a <a>verifier</a>, while
-including the refresh reference inside the <a>verifiable credential</a>
-enables both the <a>holder</a> and the <a>verifier</a> to perform future
-updates of the <a>credential</a>.
+The refresh service is only expected to be used when either the credential 
+has expired, or when the issuer does not publish credential status information. 
+It is ill advised for issuers to put the <code>refreshService</code> in a 
+<a>verifiable credential</a> that does not contain public information, 
+or whose refresh service is not protected in some way. 
```

Comment:
I think that putting lines 2214-2216 and the following sentence in a warning Note would help implementers be more aware of the risks associated with refresh services: "Placing a refresh service in a verifiable credential in such a way that it is available to a verifier could remove control and consent from the holder and essentially allow a verifiable credential about the subject to be issued directly to the verifier, bypassing the holder."

Reply:
I have had another thought. If implementors only support the Refresh Service for VPs and not for VCs, then this feature would be at risk and would be removed along with all other at risk features that do not have 2 implementations.

Reply:
If we extend the implementation spreadsheet by having two columns for the refreshService (one for refreshService in VPs, the other for refreshService in VCs), instead of the current single column, then if 2 implementations do not support it in VCs, we could remove the feature from the final version of the standard, without needed to go through another CR now.

Reply:
This is not the discussion we had during the call last week. You are suggesting something we covered, which is what @dlongley is stating above. Specifically not standardizing this makes it not possible to achieve the use case around updating public VCs provided through a digital wallet. It also doesn't prevent what you are concerned about, and makes it more likely that it'll be done w/o any way to detect it wrt. wallets warning their users that the VC may be correlatable. -1 to the suggested change. We'll have to discuss this more on another call.

Reply:
@msporny I think you are confusing two things,
i) the text of this PR, which is exactly in line with the discussion we had, so can be merged, and
ii) my suggestion about altering the implementation spreadsheet, which does not effect the text of this PR, but is certainly a discussion we can have on another call.

Reply:
Ah, ok, I thought you were arguing against your own PR.